scene/main/node.h

```diff
@@ -46,10 +46,12 @@ class Node : public Object {
 	OBJ_CATEGORY("Nodes");
 
 public:
-	enum PauseMode {
-		PAUSE_MODE_INHERIT,
-		PAUSE_MODE_STOP,
-		PAUSE_MODE_PROCESS
+	enum ProcessMode {
+		PROCESS_MODE_INHERIT, // same as parent node
+		PROCESS_MODE_NORMAL, // process only if not paused
+		PROCESS_MODE_PAUSE_ONLY, // process only if paused
+		PROCESS_MODE_ALWAYS, // process always
+		PROCESS_MODE_DISABLED, // never process
```

Comment:
Shouldn't this one be called `PROCESS_MODE_NEVER`, if it's the opposite of `ALWAYS`?

Reply:
Maybe because `Never` sounds too irreversible?
`Disabled` would suggest the possibility to (re-)activate it when requested?

Reply:
If `DISABLED` is kept, then `ALWAYS` should be changed to `ENABLED` for consistency.

Reply:
I think current naming is more intuitive.